### PR TITLE
Fix for Info page in 3.4.0

### PR DESCRIPTION
**Pull request by emikolajczak**

### CODE REVIEW

File: dark-mode.css [hunk 1]
```diff
@@ -168,7 +168,7 @@ body.dark-mode {
 /***************Main Chat*****************/
 
 body.dark-mode a {
-	color: var(--color-light-blue);
+	color: var(--info-font-color);
```
Comment (pbaity) [inline dark-mode.css:171]: Why change link color from `--color-light-blue` to `--color-gray-light`?

File: dark-mode.css [hunk 2]
```diff
@@ -530,14 +551,76 @@ body.dark-mode .rc-modal-wrapper > dialog > div {
...
+	background-color: var(--color-gray-light);
+}
+
+body.dark-mode .CodeMirror-gutter {
+	background-color: var(--color-dark);
+}
+
+body.dark-mode .setting-action {
+	border: var(--button-border-width) solid var(--info-font-color);
+	color: var(--info-font-color);
```
Comment (pbaity) [inline dark-mode.css:626]: This rule is missing a closing `}`

File: dark-mode.css [hunk 3]
```diff
@@ -530,14 +551,76 @@ body.dark-mode .rc-modal-wrapper > dialog > div {
...
+body.dark-mode .table-fake-th {
+	color: var(--info-font-color);
+}
+
+body.dark-mode .rc-input__element {
+	background-color: var(--color-dark-medium);
+	color: var(--info-font-color) !important;
+}
+
+body.dark-mode .rcx-check-box.is-focused .rcx-check-box__input:checked+.rcx-check-box__fake, .rcx-check-box.is-focused .rcx-check-box__input:indeterminate+.rcx-check-box__fake, .rcx-check-box__input:checked:focus+.rcx-check-box__fake, .rcx-check-box__input:indeterminate:focus+.rcx-check-box__fake, .rcx-radio-button.is-focused .rcx-radio-button__input:checked+.rcx-radio-button__fake, .rcx-radio-button__input:checked:focus+.rcx-radio-button__fake, .rcx-toggle-switch.is-focused .rcx-toggle-switch__input:checked+.rcx-toggle-switch__fake, .rcx-toggle-switch__input:checked:focus+.rcx-toggle-switch__fake{
```
Comment (pbaity) [inline dark-mode.css:612]: Please put these rules on separate lines to improve readability

File: dark-mode.css (hunk 1 above)
Comment (emikolajczak) [reply dark-mode.css]: This is mistake, I have fixed it to `--color-light-blue`

File: dark-mode.css (hunk 2 above)
Comment (emikolajczak) [reply dark-mode.css]: Fixed

File: dark-mode.css (hunk 3 above)
Comment (emikolajczak) [reply dark-mode.css]: fixed

**Comment by pbaity** [comment]
Thanks for the fixes @emikolajczak - when you get the chance, resolve the conflict and move the rules under the header `/**************Admin Panel******************/` (or if some rules belong elsewhere, they can go there). I wanted to help you out on this one, but I'm not 100% sure I'd make the right changes when I looked at it. 

I'm getting a 3.4.0 instance set up soon so I can test there and on my 3.2.2 snap install - as a heads up, I'm expecting that some of the rules you commented out might break dark mode in 3.2.2. But we can figure that out when we get there.

**Comment by pbaity** [comment]
@emikolajczak I went ahead and resolved the conflict and move the rules under the appropriate comment header (I think - this doesn't matter as much). I didn't change any rules, and I tested in 3.4.2 to confirm - it looks great! It does a few weird things in 3.2.2, so I created a 3.2.2 branch and release for those who are still using older versions (like me). The master branch will support the most recent version of Rocket.Chat, so I'm merging this into master now.

**Pull request merged**